<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formRegistrarTrabajador</class>
 <widget class="QDialog" name="formRegistrarTrabajador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#formRegistrarTrabajador{
	background-image: url(:/recur/imagenes/fondo_wasa.png);
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>461</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1" colspan="3">
     <widget class="QLineEdit" name="getDni">
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>DNI :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Tipo trabajador:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Sueldo:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Edad :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Codigo :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nombres :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Apellidos :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="getApellidos"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Genero :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="getCodigo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="bBMasculino">
      <property name="text">
       <string>Masculino</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QRadioButton" name="rBFemenino">
      <property name="text">
       <string>Femenino</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QSpinBox" name="getEdad"/>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="getNombres"/>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QComboBox" name="getTipoTra">
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cocinero</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QDoubleSpinBox" name="getSueldo">
      <property name="minimum">
       <double>800.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/recur/imagenes/281776-200.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pBRegistrar">
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBCerrar">
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
